src/main.rs

```diff
@@ -9,14 +9,14 @@ use anyhow::{anyhow, Context, Error};
 use cargo::util::config::Config;
 use cargo_lock::package::{Name, Package, SourceId, Version};
 use cargo_lock::Lockfile;
+use difference::{Changeset, Difference};
 use either::Either;
 use git2::{Object, ObjectType, Oid, Repository};
 use itertools::{EitherOrBoth, Itertools};
+use sources::Resolver;
```

Comment:
That's what I don't like about the idea's auto-formatter. `rustfmt` leaves it where it should be. This is our own crate's type, so it belongs to the third block O:-).